input/blog/archive.cshtml

```diff
@@ -1,5 +1,5 @@
 ---
-Enumerate: => Inputs.FilterSources("blog/posts/*").Select(x => x.GetDateTime("Published").ToString("yyyy/MM")).Distinct()
+Enumerate: => Parent.Inputs.FilterSources("blog/posts/*").Select(x => x.GetDateTime("Published").ToString("yyyy/MM")).Distinct()
```

Comment:
This is really the only substantive change - the module that handles enumeration is now inside another module so we have to climb out a level to use this little trick now.